src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -977,6 +978,12 @@ private function registerSerializerConfiguration(array $config, ContainerBuilder
             $definition->addTag('serializer.normalizer', array('priority' => -900));
         }
 
+        if (class_exists(YamlEncoder::class)) {
```

Comment:
Due `use Symfony\Component\Serializer\Encoder\YamlEncoder;` this line will not check whether or not a class exists in the Yaml namespace